hub/models.py

```diff
@@ -1751,18 +1749,34 @@ async def refresh_many(
 
     # UI
 
+    def enable_auto_import(self) -> Union[None, int]:
+        self.auto_import_enabled = True
+        if self.automated_webhooks:
+            self.refresh_webhooks()
+            # And schedule a cron to keep doing it
+            refresh_webhooks.defer(
+                external_data_source_id=str(self.id),
+            )
+        self.save()
+
+    def disable_auto_import(self):
+        self.auto_import_enabled = False
+        self.save()
+        if self.automated_webhooks and hasattr(self, "teardown_webhooks"):
+            self.teardown_webhooks()
```

Comment:
Wonder if we rename this to `teardown_unused_webhooks`. At first glance this looks like it'd also teardown auto-update webhooks, and I had to look at the function's code to understand you had covered this base.